### PR TITLE
Updated game list and refactored to use Steam IDs rather than titles

### PR DESCRIPTION
**Pull request by arcataroger**
Hi there!

I've been using your extension for a while now and found it very helpful. Thank you for this!

It looks like the games list hasn't been updated in a while. I updated them with the latest titles, and also refactored the code to use Steam IDs instead of game titles. It's more precise this way and less error-prone. I get the IDs from the official GFN games page API. The details are in this file: https://github.com/arcataroger/show-geforce-on-steam/blob/dev/getSteamIds.js

Later, I want to add auto-update functionality too, but need to first learn more about how Chrome extensions can cache files (or not). I might also want to proxy the GFN API behind something like Cloudflare so we're not hitting Nvidia directly, or at least make sure that each extension user is rate-limited and caches their own results.

Maybe a better way to do this would be to use a Github Action to update this master games list every X hours (or every GFN Thursday, for example)... I'll also investigate that later.

For now though, this at least adds a bunch of games from the last few months.

Thanks again for this extension!

### CODE REVIEW

File: contentScript.js [hunk 1]
```diff
@@ -1,255 +1,242 @@
 function page_constructor() {
-  String.prototype.clearText = function () {
-    return this.replace("’", "'")
-      .replace("–", "-")
-      .replace("®", "")
-      .replace("©", "")
-      .replace("™", "");
-  };
```
Comment (arcataroger) [inline contentScript.js:8]: By using steam IDs instead of titles, we can skip the title string comparisons

File: contentScript.js [hunk 2]
```diff
@@ -1,255 +1,242 @@
...
+
+    return Object.freeze({
+        nodeObserver,
+    });
+}
+
+const isSteamIdOnGeForceNow = gameNode => {
+    const steamID = gameNode.getAttribute('data-ds-appid') // Get the game's Steam ID from its data attribute
+    return steamIdsOnGeForceNow.has(steamID); // Check it against the list of known IDs
 }
```
Comment (arcataroger) [inline contentScript.js]: The new filter function that checks games against the list of known IDs on GFN

File: contentScript.js [hunk 3]
```diff
@@ -1,255 +1,242 @@
...
-    });
-  }
-  nodeObserver(document.querySelector(".tabarea"), insertBanner);
+    const {nodeObserver} = page_constructor();
+    //create html element that will be injected to page
+    const span = "<span class='geforce-button'>GeforceNow</span>";
+
+    function insertBanner() {
+        [...document.querySelectorAll(".tab_item")]
+            .filter(isSteamIdOnGeForceNow)
```
Comment (arcataroger) [inline contentScript.js:43]: example usage of the new filter

File: getSteamIds.js
Comment (arcataroger) [inline getSteamIds.js:1]: This file isn't used in the actual extension yet, but shows how we get the steam IDs from the GFN games page API. We can use it later to add auto-updating functionality

**Review by mefaba — APPROVED** [review]

File: getSteamIds.js
Comment (mefaba) [reply getSteamIds.js:1]: Only downside of this that, we would cause an API call every time user enters the steam.

**Pull request merged**